docs/migration.txt

```diff
@@ -153,3 +153,10 @@ on the FilterSet:
 
 To fix this, simply remove the methods from your class. You can subclass
 ``OrderingFilter`` to migrate any custom logic.
+
+
+Deprecated ``FILTERS_HELP_TEXT_FILTER`` and ``FILTERS_HELP_TEXT_EXCLUDE`` settings
+----------------------------------------------------------------------------------
+Details: https://github.com/carltongibson/django-filter/pull/484
```

Comment:
There is no details .What I missing? I use this settings in my project (as a author of patch which introduce them). See #136, #148, #292 for user feedback.